src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
```

Comment:
Here, your random instance is created every time the method is called. random - is our dependency, let's initialize dependencies in the class fields. Also, as stated in common mistakes, it's better to use name() method in order to get string representation of the enums